_posts/2017-11-17-setting-up-dotnet-core.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Setting up .NET Core
+author: Evan Kaiser
+tags:
+  - .NET Core
+  - Setup Scripts
+---
+
+## .NET Core
+
+This is an ongoing series on .NET Core.
+
+1) Setting up .NET Core
```

Comment:
not sure if just when running but the list shows up on one line in the rendered output.